utils/types.go

```diff
@@ -97,6 +97,10 @@ type KongClientConfig struct {
 	Timeout int
 
 	CookieJarPath string
+
+	TLSClientCert string
```

Comment:
Unnecessary string conversions. Please change both of these types to `[]byte` and update the code accordingly.

Reply:
Will do.

Reply:
That should have been a []byte as well but oh well, no biggie.